scripts/api_tests_generator/preprod_cases/invoices.csv

```diff
@@ -0,0 +1,69 @@
+endpoint,entity_ref,json_locator,test_purpose
```

Comment:
Could these be converted to the same dot notation as the other tests (e.g. deputy_warnings.csv)?

Reply:
Yeah good call. It's so permissive I hadn't noticed I'd missed some so good catch as it still generated the files fine :)